lib/CryptoNoteConfig.h

```diff
@@ -60,6 +60,8 @@ const uint64_t MINIMUM_FEE_V2                                 = UINT64_C(1000000
 const uint32_t MINIMUM_FEE_V2_HEIGHT                          = 800000;
 const uint64_t MINIMUM_FEE                                    = MINIMUM_FEE_V2;
 const uint64_t MAXIMUM_FEE                                    = UINT64_C(100000000);
+const uint32_t REMOTE_NODE_FEE_FACTOR                         = 0.25; // percent
```

Comment:
This is an implicit conversion from `double` to `uint32_t`.
Value will be implicitly converted from `0.25` to `0`.
@nnian Are you sure that it is working as it should?